doc/SpecifyingApplications/CreatingProfileInstanceList/CreatingProfileInstanceList.md

```diff
@@ -30,9 +30,8 @@ This is a step by step cookbook for creating the ProfileInstanceList.
 * The UUID shall follow the [Structure of UUIDs](../../ElementsApplicationPattern/Names/StructureOfUuids/StructureOfUuids.md).  
 * The LayerID shall be set on "action".  
 * The ObjectType shall be set on "p" (Profile).  
-* The ApiSegment shall be set on "1" (management, which is individual to this application).  
-* The ApplicationNumber shall be set on "0" (the application itself).  
-* The SequenceNumber shall start at "00" and must be unique within the scope of instances of Profiles with the same LayerID.  
+* The ApiSegment shall be set on "mi" (management, which is individual to this application).  
```

Comment:
As per the Structure of UUIDs diagram(uuid-structure.png) and the concept under the ApiSegment , “UUIDs of Profile, ForwardingDomain and Link don't identify an ApiSegment.”
In action profile , 
The ApiSegment shall be set on "mi" (management, which is individual to this application). 

This statement is contradicting with the concept in the Structure of uuid

Either the uuid-structure needs to be corrected 
Or
This line should be deleted

Reply:
Deleted this line.